tests/src/test_projects/storagemapvec/tests/clear.rs

```diff
@@ -0,0 +1,36 @@
+use super::utils::{
+    abi_calls::{
+        push,
+        clear,
+        len,
+        is_empty,
+    }, 
+    test_helpers::{
+      setup,  
+    },    
+};
+
+#[tokio::test]
+pub async fn can_clear() {
+    let instance = setup().await;
+
+    push(&instance, 1, 50).await;
+    push(&instance, 1, 100).await;
+    push(&instance, 1, 150).await;
+
+    push(&instance, 2, 50).await;
```

Comment:
Maybe change the 50, 100, and 150 to other values for the second key so we can ensure that the one key and two key return their respective values